src/adapters/mockData.js

```diff
@@ -0,0 +1,86 @@
+},
+{
+  deviceId: 'fd8f12fdced8098aaac31836c8b98960727060b57d48148e15cc34ad4ba1394a',
+  kind: 'audioinput',
+  label: 'Microphone Array (Realtek(R) Audio)',
+  groupId: '9492a44c685660ca64ad1340d0849f133193263fd3ce2255282973a695721d10',
+},
+{
+  deviceId: 'df434123000a75a161b1841b08f7318617b419aae3c93683b0fcb3389470b39a',
+  kind: 'audioinput',
+  label: 'Microphone (Realtek USB2.0 Audio) (0bda:402e)',
+  groupId: 'c7dfb4decc063e7e7487c56d6bdccea252c76266673bb10a1276af1443463ede',
+},
+{
+  deviceId: '79a2df2a81176acde237e064a9b213cd1bd32608106bd4ee5c30242eee01945f',
+  kind: 'audioinput',
+  label: 'Headset Microphone (Jabra EVOLVE 20 SE MS) (0b0e:0300)',
+  groupId: '2601739aa80181d7af9e62971342c0303e6d38b39686d70c6e8d351b686a02d5',
+},
+{
+  deviceId: 'f4491e7c9ad16139cc485d99e39234313172be31fc00c086acbdecb21236ccf6',
+  kind: 'audioinput',
+  label: 'Microphone (HD Webcam C525) (046d:0826)',
+  groupId: '7bc852104102701275b0eddcf7ce237ebb0bee1b65ded7d24be4b5d62c57a6bc',
+},
+{
+  deviceId: '2a9f83242466302e2130134a57162f3562c59bd9ea34daa7f6fc2ad43a29265b',
+  kind: 'videoinput',
+  label: 'Logitech HD Webcam C525 (046d:0826)',
+  groupId: '54d845f717574723b1bf5d552d72a6594b3bed860bb82b60a1e432cf0e5df1c2',
+},
+{
+  deviceId: 'default',
+  kind: 'audiooutput',
+  label: 'Default - Headset Earphone (Jabra EVOLVE 20 SE MS) (0b0e:0300)',
+  groupId: '2601739aa80181d7af9e62971342c0303e6d38b39686d70c6e8d351b686a02d5',
+},
+{
+  deviceId: 'communications',
+  kind: 'audiooutput',
+  label: 'Communications - Headset Earphone (Jabra EVOLVE 20 SE MS) (0b0e:0300)',
+  groupId: '2601739aa80181d7af9e62971342c0303e6d38b39686d70c6e8d351b686a02d5',
+},
+{
+  deviceId: '5e2cade11fab305ca3773e507b06e60d0a65ed8d6a19da2927a287c70e713dc7',
+  kind: 'audiooutput',
+  label: 'Line (Realtek USB2.0 Audio) (0bda:402e)',
+  groupId: 'c7dfb4decc063e7e7487c56d6bdccea252c76266673bb10a1276af1443463ede',
+},
+{
+  deviceId: '91e9c4b27e0b7bc8f41f3076a66e2968d4e229a5f388c2fe9f251bf8d54d7a34',
+  kind: 'audiooutput',
+  label: 'Headphones (Realtek USB2.0 Audio) (0bda:402e)',
+  groupId: '9f0a561c2cc00e6bb2e6413e2af3d9e5f21db654e2b9776b79079a2aced25470',
+},
+{
+  deviceId: 'a2f1a439c64a73b712a54c16f77716fb6040d87de312d211ef886944877ecea2',
+  kind: 'audiooutput',
+  label: 'Speakers (Realtek(R) Audio)',
+  groupId: '9492a44c685660ca64ad1340d0849f133193263fd3ce2255282973a695721d10',
+},
+{
+  deviceId: 'c2fcaf0c6b0bc7adc1192ba0b2dd236f7926e2ae163c56f80fa51613f9b9ec77',
+  kind: 'videoinput',
+  label: 'Integrated Camera (04f2:b6d9)',
+  groupId: '7aac3778d2d088d56ead222659241fdbb599837e2aa71ff4ceced4536bb8f259',
+},
+{
+  deviceId: '85e52fb55424206524719fab873ea7c3419c496ce1f691a009066ada5feaf813',
+  kind: 'audiooutput',
+  label: 'Headset Earphone (Jabra EVOLVE 20 SE MS) (0b0e:0300)',
+  groupId: '2601739aa80181d7af9e62971342c0303e6d38b39686d70c6e8d351b686a02d5',
+}];
+
+export default mockDevices;
```

Comment:
maybe `export default {mockDevices}`, so that other data can be added in the future.